packages/shared/src/utils/config/config.ts

```diff
@@ -52,31 +52,29 @@ export const getAppId = () => {
     const current_domain = getCurrentProductionDomain() || '';
     window.localStorage.removeItem('config.platform'); // Remove config stored in localstorage if there's any.
     const platform = window.sessionStorage.getItem('config.platform');
-    const { is_pathname_bot, is_config_route_bot } = isBot();
+    const domain_app_id = domain_app_ids[current_domain as keyof typeof domain_app_ids];
```

Comment:
@rupato-deriv do we need the `typeof` in this assertion?

Reply:
@shafin-deriv this is not my change